packages/technical-features/application/agnostic/src/start-application/start-application.injectable.ts

```diff
@@ -1,19 +1,19 @@
+/* eslint-disable prettier/prettier */
```

Comment:
No, we either use the prettier or we don't. Let's not start disabling it file to file.

Reply:
Also, this doesn't have anything to do with the contents of this PR. Revert.